components/clarinet-deployments/src/lib.rs

```diff
@@ -106,6 +106,9 @@ fn update_session_with_genesis_accounts(
     deployment: &DeploymentSpecification,
 ) {
     if let Some(ref spec) = deployment.genesis {
+        let addresses: Vec<_> = spec.wallets.iter().map(|w| w.address.clone()).collect();
+        session.interpreter.save_genesis_accounts(addresses);
```

Comment:
`genesis_accounts` are saved in the store. It allows to skip remote_data fetching for those addresses (big perf gain on session start)